server/src/server.ts

```diff
@@ -1,15 +1,19 @@
+import * as dotenv from "dotenv";
 import express, { Application, Request, Response } from "express";
 import cors from "cors";
 
 const app: Application = express();
 app.use(express.json());
 app.use(cors());
-const port = 3000;
+
+dotenv.config();
+const hostname = process.env.HOST;
+const port = process.env.PORT;
```

Comment:
Nitpick: a useful thing to have is a single file/module that loads all of the environment variables and is the only one that does it. Then it can be imported into all of the modules that need it. It keeps the variables more organized and easier to keep track of their usage. Also, it allows a cleaner way to apply defaults. Something like this:
```js
export default {
  server: {
    hostname: process.env.HOST,
    port: process.env.PORT || 3000
  }
};
```

Reply:
I know what you are talking about. Wanted to do that but since I'm using them in only one place I decided to do it as simple as it gets. My plan was to add that in the future as a small PR before adding authentication where I will need additional environment variables and will use them in multiple places. Will address that now.